data-management/data-interface.js

```diff
@@ -249,19 +258,8 @@ const editUser = async (parameters, context) => {
 
 const updateMyUser = async (parameters, context) => {
     formatParams(parameters);
-    try {
-        let userInfo = context.userInfo;
-        if (!userInfo) {
-            return new Error(errorName.NOT_LOGGED_IN);
-        }
-        else{
-            parameters = {...userInfo, ...parameters.userInfo};
-            parameters.editDate = (new Date()).toString()
-            return await neo4j.updateMyUser(parameters, userInfo);
-        }
-    } catch (err) {
-        return err;
-    }
+    validator.isValidLoginOrThrow(context.userInfo);
+    return await neo4j.updateMyUser({...parameters.userInfo}, {...context.userInfo});

```

Comment:
1 . removed a user object
2. bug fix; session info will store if no parameters attached
3. removed unnecessary catch and block